config/samples/ndb_v1alpha1_database.yaml

```diff
@@ -18,3 +18,14 @@ spec:
     size: 10
     timezone: "UTC"
     type: postgres
+    profiles:
+      compute:
+        id: ""
+      software:
+        name: ""
+        id: ""
+      network:
+        name: ""
+      dbParam:
+        name: ""
+        id: ""
```

Comment:
Can we also add the dummy values in the example? Like "Software Profile Name", "Network Profile Id" etc

Reply:
I am actually refraining from adding it here. We can discuss this offline.